### PR TITLE
Update dbt for Action Network

### PR DESCRIPTION
**Pull request by kanelouise**
Per [this card](https://techallies.atlassian.net/browse/DE-241)

@emily-flambe, thanks for testing dbt. lmk what running dbt locally on your machine reveals.

### CODE REVIEW

**Review by jitoquinto — COMMENTED** [review]
Oh wow this A LOT of tables. I didnt look through all the files but overall the changes make sense to me! Quick question on the matviews though, are the from statements supposed to be 
```
from {{ source('cta','base_table_name') }}
```
instead of `ref`?

**Comment by emily-flambe** [comment]
> Oh wow this A LOT of tables. I didnt look through all the files but overall the changes make sense to me! Quick question on the matviews though, are the from statements supposed to be
> 
> ```
> from {{ source('cta','base_table_name') }}
> ```
> 
> instead of `ref`?

@jitoquinto this was something @kanelouise and I discussed outside this PR - in general I propose we switch to using `ref` instead of `source` in our matview queries. Either one technically works, but with `ref` you don't need to add the tables to the `sources.yml` file - so it's easier to use without potentially hitting errors because a source was left out of sources.yml. Open to opinions if that seems like a bad idea though!

**Comment by jitoquinto** [comment]
> > Oh wow this A LOT of tables. I didnt look through all the files but overall the changes make sense to me! Quick question on the matviews though, are the from statements supposed to be
> > ```
> > from {{ source('cta','base_table_name') }}
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > instead of `ref`?
> 
> @jitoquinto this was something @kanelouise and I discussed outside this PR - in general I propose we switch to using `ref` instead of `source` in our matview queries. Either one technically works, but with `ref` you don't need to add the tables to the `sources.yml` file - so it's easier to use without potentially hitting errors because a source was left out of sources.yml. Open to opinions if that seems like a bad idea though!

ohh, i just thought it didnt work cause they arent ran together (different dbt runs right?) but if it works im good with it 👍

**Comment by emily-flambe** [comment]
> > > Oh wow this A LOT of tables. I didnt look through all the files but overall the changes make sense to me! Quick question on the matviews though, are the from statements supposed to be
> > > ```
> > > from {{ source('cta','base_table_name') }}
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > instead of `ref`?
> > 
> > 
> > @jitoquinto this was something @kanelouise and I discussed outside this PR - in general I propose we switch to using `ref` instead of `source` in our matview queries. Either one technically works, but with `ref` you don't need to add the tables to the `sources.yml` file - so it's easier to use without potentially hitting errors because a source was left out of sources.yml. Open to opinions if that seems like a bad idea though!
> 
> ohh, i just thought it didnt work cause they arent ran together (different dbt runs right?) but if it works im good with it 👍

lol I assumed it would work, definitely should test to make sure!

File: dbt-cta/action_network/models/sources.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 version: 2
 sources:
-- name: cta
-  database: "{{ env_var('CTA_PROJECT_ID') }}"
-  schema: "{{ env_var('CTA_DATASET_ID') }}"
+- name: tmc_action_network
```
[the remark this answers is not in the document]
Comment (kanelouise) [reply dbt-cta/action_network/models/sources.yml]: Updated!

**Comment by kanelouise** [comment]
> > > > Oh wow this A LOT of tables. I didnt look through all the files but overall the changes make sense to me! Quick question on the matviews though, are the from statements supposed to be
> > > > ```
> > > > from {{ source('cta','base_table_name') }}
> > > > ```
> > > > 
> > > > 
> > > >     
> > > >       
> > > >     
> > > > 
> > > >       
> > > >     
> > > > 
> > > >     
> > > >   
> > > > instead of `ref`?
> > > 
> > > 
> > > @jitoquinto this was something @kanelouise and I discussed outside this PR - in general I propose we switch to using `ref` instead of `source` in our matview queries. Either one technically works, but with `ref` you don't need to add the tables to the `sources.yml` file - so it's easier to use without potentially hitting errors because a source was left out of sources.yml. Open to opinions if that seems like a bad idea though!
> > 
> > 
> > ohh, i just thought it didnt work cause they arent ran together (different dbt runs right?) but if it works im good with it 👍
> 
> lol I assumed it would work, definitely should test to make sure!

Luckily, it did indeed work!

**Review by jitoquinto — APPROVED** [review]
sweet!

**Pull request merged**